source/js/main.js

```diff
@@ -20,6 +20,21 @@ window.addEventListener('DOMContentLoaded', () => {
   });
 });
 
+var navMain = document.querySelector('.page-nav');
+var navToggle = document.querySelector('.page-nav__toggle');
+
+navMain.classList.remove('page-nav--nojs');
+
+navToggle.addEventListener('click', function() {
+  if (navMain.classList.contains('page-nav--closed')) {
+    navMain.classList.remove('page-nav--closed');
+    navMain.classList.add('page-nav--opened');
+  } else {
+    navMain.classList.add('page-nav--closed');
+    navMain.classList.remove('page-nav--opened');
+  }
```

Comment:
Это всё можно упростить до двух строчек
```suggestion
   navMain.classList.toggle('page-nav--closed');
   navMain.classList.toggle('page-nav--opened');
```

Reply:
Так же можно добавить немного доступности.
```html
 <!-- Даём id, чтобы потом написать каким элементом управляет кнопка-->
<nav class="page-nav page-nav--closed page-nav--nojs" id="main-menu">
 <!-- твой код -->
</nav>
 <!-- Зря не послушал меня, что проще сделать две кнопки
 Даём подпись лейблом, чтобы проще менять. А так же состояние открытия, и говорим каким элементом управляет кнопка
 -->
<button class="page-nav__toggle" type="button" aria-label="Открыть меню" aria-expanded="false" aria-controls="main-menu">
</button>
```

В скрипте:
```js
  const shouldBeOpened = navToggle.ariaExpanded === 'false';  // Определяем стоит ли открыть
  document.body.style.overflow = shouldBeOpened ? 'hidden' : '';  // не позволяет скроллить
  navToggle.ariaExpanded = String(shouldBeOpened); // говорим что кнопка открытка
  navToggle.ariaLabel = `${shouldBeOpened ? 'Открыть' : 'Закрыть'} меню`; // меняем подпись
```

Reply:
Обрати внимание, что шаблон советует модульность
![image](https://user-images.githubusercontent.com/47776594/179061318-b646015c-196e-4e8b-ac41-7e4c96880e1e.png)
![image](https://user-images.githubusercontent.com/47776594/179061700-ffbad05c-2b5b-44d1-86d6-f32241654d70.png)